### PR TITLE
Update purge doc

### PR DESCRIPTION
**Pull request by MichaelTsengLZ**

### CODE REVIEW

**Comment by PRMerger3** [comment]
@MichaelTsengLZ : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
#label:"aq-pr-triaged"

File: data-explorer/kusto/concepts/data-purge.md [hunk 1]
```diff
@@ -116,8 +116,8 @@ Purge command may be invoked in two ways for differing usage scenarios:
 > To connect to a cluster using the Azure Data Explorer Web UI, see [Add clusters](../../web-query-data.md#add-clusters).
 	 
   ```kusto
-	 // Connect to the Data Management service - this command only works in Kusto.Explorer
-	 #connect "https://ingest-[YourClusterName].[region].kusto.windows.net" 
+	 // Connect to the Data Management service - this command only works in Kusto.Explorer without [database].
+	 #connect "https://ingest-[YourClusterName].[region].kusto.windows.net/[database]"
```
Comment (vladikbr) [inline data-explorer/kusto/concepts/data-purge.md:120]: /[database] is not really needed, as far as I can see - it works both in KE and in KWE without the database.
So the "- this command only works in Kusto.Explorer without [database]" part can also be deleted

**Comment by shsagir** [comment]
@MichaelTsengLZ Per @vladikbr's comment, the proposed changes are not required and will not be applied. Hence, we are closing this issue. Thanks for your feedback, we really appreciate your input to improve our documentation.

#please-close

**Pull request closed without merging**